src/BlazorWebView/src/WindowsForms/BlazorWebView.cs

```diff
@@ -175,17 +175,17 @@ private void HandleRootComponentsCollectionChanged(object sender, NotifyCollecti
         /// <inheritdoc />
         protected override void Dispose(bool disposing)
         {
-            base.Dispose(disposing);
-
             if (disposing)
             {
-                _webviewManager?.Dispose();
-                _webview?.Dispose();
+				// Dispose this component's contents before calling base.Dispose() because that will dispose the WebView2 control, likely
+				// preventing user-written disposal logic from working (because it might try to use Blazor stuff, which wouldn't work anymore).
```

Comment:
The comment doesn't explain why it's no longer necessary to call `_webview?.Dispose()`. Could you clarify?

Reply:
It does sort of say it. It says that base.Dispose() will dispose it, so we don't need to. (The problem was two-fold: 1. We were doing unnecessary work; 2. We did the unnecessary work in a broken way.)